src/views/home/components/transaction-form/transaction-form.view.tsx

```diff
@@ -134,8 +120,22 @@ const TransactionForm: FC = () => {
         </div>
       </Card>
       <div className={classes.button}>
-        <Button type="submit">Continue</Button>
+        <Button type="submit" disabled={isInvalid}>
+          Continue
+        </Button>
+        {isInvalid && !localTransaction.amount.isZero() && (
+          <Typography type="body1" className={classes.error}>
+            Insufficient balance
+          </Typography>
+        )}
       </div>
+      {list && (
+        <List
+          placeholder={list.type === "chain" ? "Search network" : "Search token"}
+          list={list}
+          onClose={() => setList(undefined)}
+        />
```

Comment:
The filter of the `List` is now broken.

![filter-broken](https://user-images.githubusercontent.com/15896177/163179263-9466ed2f-04b7-4dd1-bdf8-950f811526cd.gif)